tough/src/editor/mod.rs

```diff
@@ -74,6 +74,18 @@ impl RepositoryEditor {
         let root_buf_len = root_buf.len() as u64;
         let root = serde_json::from_slice::<Signed<Root>>(&root_buf)
             .context(error::FileParseJson { path: root_path })?;
+
+        // Sanity check of root
```

Comment:
https://gist.github.com/seanmhanson/fe370c2d8bd2b3228680e38899baf5cc

```suggestion
        // Quick check that root is signed by enough key IDs
```